### PR TITLE
STIJ-3: Added search to header

### PR DESCRIPTION
**Pull request by meirege**

### CODE REVIEW

File: components/21-atoms/button-submit/button-submit.twig [hunk 1]
```diff
@@ -0,0 +1 @@
+<button type="submit" class="{{ modifier }}">{{ text }}</i></button>
```
Comment (Jeroen005) [inline components/21-atoms/button-submit/button-submit.twig]: Maybe we should move the icon class to the button element.

Comment (meirege) [reply components/21-atoms/button-submit/button-submit.twig]: Done.

File: public/styleguide/js/header.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+(function ($) {
```
Comment (Jeroen005) [inline public/styleguide/js/header.js]: This one should be gitignored.

Comment (meirege) [reply public/styleguide/js/header.js]: Why?

Comment (Jeroen005) [reply public/styleguide/js/header.js]: `components/41-organisms/header/header.js` and `public/styleguide/js/header.js` are the same. `public/styleguide/js/header.js` is generated with gulp, or am I wrong? :)

Comment (meirege) [reply public/styleguide/js/header.js]: It gets copied by gulp to the public and build directories, yes.

Comment (Jeroen005) [reply public/styleguide/js/header.js]: File is still available in git.

**Pull request merged**